recipes/configs/qwen2/0.5B_lora_single_device.yaml

```diff
@@ -19,11 +19,10 @@
 # Model Arguments
 model:
   _component_: torchtune.models.qwen2.lora_qwen2_0_5b
-  lora_attn_modules: ['q_proj', 'v_proj']
+  lora_attn_modules: ['q_proj', 'k_proj', 'v_proj']
   apply_lora_to_mlp: False
-  apply_lora_to_output: False
-  lora_rank: 8
-  lora_alpha: 16
+  lora_rank: 64
+  lora_alpha: 32
```

Comment:
it should be the opposite :P 

alpha is double of rank. Check llama configs for example 
![image](https://github.com/user-attachments/assets/3dd6cb89-d4db-4e0e-be70-170f4fa744ae)

btw, I think that having LoRA on the MLP is better than adding 'k_proj'.

Reply:
I don't think this is consistent across our configs. For example, Gemma2B has:

```
model:
  _component_: torchtune.models.gemma.lora_gemma_2b
  lora_attn_modules: ['q_proj', 'k_proj', 'v_proj']
  apply_lora_to_mlp: True
  lora_rank: 64
  lora_alpha: 16
```

I think you're right though. Happy to change this

Reply:
note: `apply_lora_to_mlp` did impact training speed/mem usage for me without significantly improving loss

Reply:
Ah misspoke - definitely go with Felipe's suggestion. Alpha being double rank.